portalnet/src/overlay_service.rs

```diff
@@ -818,6 +818,9 @@ where
                             Some(enr) => enr,
                             _ => {
                                 warn!("Received uTP payload from unknown {peer}");
+                                if let Some(responder) = callback {
+                                    let _ = responder.send((None, true, query_info.trace));
+                                };
                                 return;
```

Comment:
Nice catch! It looks like we will also want to do something similar on line 845-ish & 861-ish?

Reply:
And I checked inside the utp functionality over in `overlay.rs`, it looks like this case is handled there, but might be worth double-checking my checking.